src/main/cpp/t06_longest.cpp

```diff
@@ -22,5 +22,30 @@
 using namespace std;
 
 int t06_longest() {
+    char c;
+    int n = 1;
+    int max = 1;
+    string maxW = "";
+    string w = "";
+
+    while(cin.get(c)){
+        w += c;
+        if(int(c) == 32){
+            if(n > max){
+                max = n;
+                maxW = w;
+            }
+            n = 0;
+            w="";
+        }
+        n++;
+    }
+
+    if(n > max){
+        max = n;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'max' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4678874)